presto-main/src/main/java/io/prestosql/sql/analyzer/FeaturesConfig.java

```diff
@@ -131,6 +134,7 @@
     private boolean enableDynamicFiltering = true;
     private int dynamicFilteringMaxPerDriverRowCount = 100;
     private DataSize dynamicFilteringMaxPerDriverSize = DataSize.of(10, KILOBYTE);
+    private Duration dynamicFilteringRefreshInterval = new Duration(100, MILLISECONDS);
```

Comment:
What was the default previously? Arent we changing rate too aggressively?

Reply:
The earlier default was 1 sec. It might be worth checking the time taken by collectDynamicFilters when there are large no. of queries active with multiple stages containing dynamic filters.

Reply:
I've changed it to 200ms. I don't think it's too aggressive. Anyway, it's limited to single thread.